### DISCUSSION TITLE
How to Skip Loading

### DISCUSSION THREAD

**Discussion by bwkam** [KadeArchive/Kade-Engine#2501] [Q&A]
Hello, so I'm currently making my mod, and doing a good progress, but there is a problem, I would like to skip that loading screen
so that whenever anyone downloads my mod goes directly to main menu, so how can I delete this loading screen without causing
any wrong stuff or errors in the code?

**Comment by SuxxorState** [comment]
Hop on to your designated code engine thingamabob, and open titlestate.hx
find the if polymod code at around line 50-60, and replace that function with this:
FlxG.switchState(new MainMenuState());

then open up mainmenustate.hx

find:

			if (controls.BACK)
			{
				FlxG.switchState(new TitleState());
			}

and delete it.

and boom! the title is skipped entirely. im sure you can delete most of the other lines of code in titlestate.hx, but thats for you to mess around with lol

**Reply by SuxxorState** [reply]
you cant skip the caching part of the loading screen though
otherwise that's how ya do it
